content/basics/ably.textile

```diff
@@ -0,0 +1,92 @@
+
+Read more in our "Presence and occupancy section":/presence-occupancy.
+
+h3(#message-storage-and-history). Message storage and history
+
+By default messages are only stored for two minutes on Ably’s servers, these messages are retrieved automatically by the SDK in the event of a lost connection. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the History API to retrieve messages.
+
+Ably's message storage and history allows messages to be accessed after initially being sent. You can persist the last message sent using the rewind channel option or enable persistence of all messages for 24 hours for the free tier or 72 hours for paid tiers. These messages can be retrieved using the History API. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations.
+
+Read more in our "Message storage and history section":/storage-history.
+
+h3(#push-notifications). Push notifications
+
+Ably's Push Notifications feature allows messages to be pushed to subscribers without needing a device to maintain a connection to Ably. Push Notifications are commonly used to display visual notifications to users or launch background processes for apps in a battery-efficient manner.
+
+Read more in our "Push Notifications section":/push.
+
+h2(#infrastructure). About the infrastructure
+
+To ensure unrivaled realtime messaging and data streaming performance, availability, and reliability, the Ably Global Network has 7 geographically-distributed core routing datacenters and 307 edge acceleration points-of-presence(PoPs).
+
+Building on the Global Network Ably focuses on a seamless experience, that's why Ably's infrastructure is built around the "Four Pillars of Dependability":https://ably.com/four-pillars-of-dependability.
+
+h3(#performance). Performance
+
+By providing consistently minimized latency and bandwidth requirements, developers know what to expect and to design towards. If you know the boundaries Ably will always operate within, you can design, build, and scale features around them, confident they’ll perform as expected under various conditions.
+
+h3(#integrity). Integrity
+
+Knowing that many apps rely on a strict sequence of messages that depend on each other, Ably is designed to maintain the end-to-end integrity of messages. This allows for simplified app architecture as you don't need to develop around missed, unordered, or duplicated messages. Integrity is guaranteed at scale thanks to Ably's modern system architecture.
+
+h3(#reliability). Reliability
+
+Being able to trust your provider is one of the most important aspects of choosing a provider. Ably ensures this trust with a platform designed with redundancy at a regional and global level based on statistical risks of failure, continuing even in the event of multiple infrastructure failures.
```

Comment:
First sentence is redundant. Can we name what type of provider we mean here? That might help.